### ISSUE TITLE
Need ability to explicitly define int field values passed to #write_point(s)

### ISSUE THREAD

**Issue by jimryan** [influxdata/influxdb-ruby#120]
The line protocol now requires that int's be explicitly defined via a trailing 'i' (see https://github.com/influxdb/influxdb/pull/3526).

It's currently impossible to accomplish this using the `#write_point`/`#write_points` methods, so you'll wind up with an error like this:

    InfluxDB::Error: write failed: field type conflict: input field "foo" on measurement "bar" is type float64, already exists as type integer

I'm happy to work up a PR, but I'm not sure what the best approach is.  I don't know that we can simply tack on an 'i' for all `Fixnums` without breaking things for users currently passing them to `float` fields.  

Right now, my temporary workaround is to add a wrapper class around `Fixnum` that tacks on an 'i' in its `#to_s` method, which I then pass to `#write_point`, but that's not a longterm/ideal solution, unless it's exposed by the gem itself.

**Comment by nlsrchtr** [comment]
:+1:

**Comment by dmke** [comment]
I've run into the same issue. Based on the suggestion by @jimryan, I currently use this workaround:

```ruby
class InfluxDB::Int64 < Struct.new(:to_i)
  def to_s
    "#{to_i}i"
  end

  alias inspect to_s
end

def int64(value)
  InfluxDB::Int64.new value.to_i
end

# ...

client.write_point series: "foo", values: { bar: int64(42) } 
```

But I'm all for a generic "i" suffix for Integers. This would be consistent with the other official client implementations:

- [Python](https://github.com/influxdata/influxdb-python/blob/3390985/influxdb/line_protocol.py#L62-L63)
- [PHP](https://github.com/influxdata/influxdb-php/blob/430d3e2/src/InfluxDB/Point.php#L143-L144)
- [Java](https://github.com/influxdata/influxdb-java/blob/878358e/src/main/java/org/influxdb/dto/Point.java#L303-L304)

**Comment by toddboom** [comment]
This should be fixed now with #131, which I just merged. Let me know if additional support is needed. Thanks!
